social_distribution/app/templates/app/public_profile.html

```diff
@@ -12,7 +12,9 @@
         <div class="col-xl">
             <h3>{{ author.displayName }}'s posts</h3>
                         {% for post in posts %}
-                            {% include './post_card.html' %}
+                            {% if not post.unlisted %}
```

Comment:
this should prob be done in the view, by adding unlisted=False to the filter here: https://github.com/cmput404fa22/404-Project/blob/faffb7162dbc926751b29c63e556e213136b2831/social_distribution/app/views/user.py#L113